Makefile

```diff
@@ -30,17 +30,9 @@ TOOLS_BIN:=tools/bin
 $(TOOLS_BIN):
 	mkdir $(TOOLS_BIN)
 
-# adapted from https://stackoverflow.com/a/12099167/552902
 HUGO_OS:=Linux
-HUGO_ARCH:=32bit
+HUGO_ARCH:=64bit
 HUGO_EXT:=tar.gz
-ifeq ($(OS),Windows_NT)
-	HUGO_OS:=Windows
-	HUGO_EXT:=zip
-ifeq ($(PROCESSOR_ARCHITECTURE),AMD64)
-	HUGO_ARCH:=64bit
-endif
-endif
```

Comment:
Looks good to me, my only question is: removing these lines means people in the community using Windows will not be able to compile and contribute to the docs repo?